tests/integration/single_table/test_copulas.py

```diff
@@ -361,3 +362,35 @@ def test_numerical_columns_gets_pii():
         'numerical': {0: 21, 1: 24, 2: 22, 3: 23, 4: 22, 5: 24, 6: 23, 7: 23, 8: 24, 9: 23}
     })
     pd.testing.assert_frame_equal(expected_sampled, sampled)
+
+
+def test_categorical_column_with_numbers():
+    """Test that categorical column represented with numbers works end to end."""
+    # Setup
+    data = pd.DataFrame({
+        'category_col': [
+            1, 2, 1, 2, 1, 2, np.nan, 1, 1, np.nan, 2, 2, None, 2,
+            1, 1, None, 1, 2, 2
```

Comment:
Just so you know, pandas converts the Nones here to np.nan which is why the dtype stays float. They aren't really necessary because of that. If ypou want the `None` values to stay you have to manually set the dtype to object

Reply:
Yes I put `np.nan` and `None` now as a sanity check that everything works for both.

Reply:
No I mean that even though you put both in here, pandas converts the Nones to np.nan behind the scenes so None is not actually being tested. To test it you have to set the dtype to object